.release/docker/Makefile

```diff
@@ -20,4 +20,4 @@ docker:
 	echo "Building image for $(PROJECT_NAME):$(GITHUB_SHA)-$(VERSION)-$(ARCH)"
 	docker buildx create --use
 	docker buildx build --load --build-arg VERSION=$(VERSION) --build-arg NAME=$(PROJECT_NAME) --platform=linux/$(ARCH)$(GOARM_DOCKER) -t $(PROJECT_NAME):$(GITHUB_SHA)-$(VERSION)-$(ARCH) .
-	docker save --output $(PROJECT_NAME)_$(VERSION)_docker_linux_$(ARCH).tar $(PROJECT_NAME):$(GITHUB_SHA)-$(VERSION)-$(ARCH)
\ No newline at end of file
+	docker save --output $(PROJECT_NAME)_$(VERSION)_docker_linux_$(ARCH).tar $(PROJECT_NAME):$(GITHUB_SHA)-$(VERSION)-$(ARCH)
```

Comment:
Since it's just the addition of a newline, should we just revert this change?

Reply:
To be honest, I don't like the red circle that appears on GitHub for files w/o new lines 😅

But more seriously, your question got me curious about the context of the new line at the EOF, and why GitHub displays it. It is a POSIX standard for a text file which contain lines, and lines include the terminating newline character. So this one is considered an invalid line to POSIX purists.

[Text file](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_403):
> A file that contains characters organized into zero or more lines. The lines do not contain NUL characters and none can exceed {LINE_MAX} bytes in length, including the <newline> character. Although POSIX.1-2008 does not distinguish between text files and binary files (see the ISO C standard), many utilities only produce predictable or meaningful output when operating on text files. The standard utilities that have such restrictions always specify "text files" in their STDIN or INPUT FILES sections.

[Line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)
> A sequence of zero or more non- <newline> characters plus a terminating <newline> character.

Reply:
If you don't mind, I'd like to keep the newline to keep consistency with the rest of the files in the repo.